pkgs/development/tools/vagrant/.gitignore

```diff
@@ -0,0 +1,2 @@
+/Gemfile
+/Gemfile.lock
```

Comment:
Why would these files end up here? (Just wondering why they need to be ignored)

Reply:
with the update script this would be become unnecessary.

Reply:
Normally the bundix workflow involves copying the Gemfile and Gemfile.lock into nixpkgs, however vagrant is a bit special. Since we're using empty files for those two, copying them into nixpkgs does nothing, so this was meant to ease the workflow for someone who is used to the standard bundix workflow.

This isn't necessary, but I thought it would be nice to have.

Reply:
I think the files showing up in the diff would be enough indication. Putting files somebody *might* accidentally copy into the repo in the gitignore seems like quite a can of worms to open :P